config/jest/matchers/toHaveNoAxeViolations.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
```

Comment:
Because we leverage `babel-jest`, we should be able to use the `import` syntax for these files